src/NUnitFramework/framework/Internal/Execution/CompositeWorkItem.cs

```diff
@@ -279,8 +279,8 @@ public int Compare(WorkItem x, WorkItem y)
             {
                 var xKey = int.MaxValue;
                 var yKey = int.MaxValue;
-                if (x.Test.Properties.ContainsKey(PropertyNames.Order)) int.TryParse(x.Test.Properties[PropertyNames.Order][0].ToString(), out xKey);
-                if (y.Test.Properties.ContainsKey(PropertyNames.Order)) int.TryParse(y.Test.Properties[PropertyNames.Order][0].ToString(), out yKey);
+                if (x.Test.Properties.ContainsKey(PropertyNames.Order)) xKey=(int)x.Test.Properties[PropertyNames.Order][0];
```

Comment:
I thought that the coding standards indicated that single line if statements were to be discouraged?  Perhaps I am mistaken?